package.json

```diff
@@ -30,11 +30,13 @@
   },
   "homepage": "https://github.com/single-spa/sofe-deplanifester#readme",
   "dependencies": {
+    "@azure/storage-blob": "^12.1.2",
```

Comment:
Do both the old and new azure clients work with all versions of Azure storage? Am thinking about backwards compatibility for those already using the older azure-storage package.

Reply:
Since Azure deprecated the old package I'm not concerned also Azure storage is a SaaS solution, cant think of versioning api's there.

Reply:
👍